src/views/Task/Create.js

```diff
@@ -6,22 +6,22 @@ import moment from 'moment'
 import Alert from '../../components/Alert'
 import camelcase from 'camelcase'
 import { withRouter } from 'react-router-dom'
+import './Task.css'
 
 export default withRouter(class Create extends Component {
   state = {
     startDate: moment(),
     endDate: moment(),
     title: "",
     description: "",
-    urgency: "",
+    urgencystring: 'norush',
```

Comment:
i can see you use norush as a validater later maeby add a coment to specify what is means?

Reply:
ican see it says norush now :D cammelcase meaby?

Reply:
Will add explanatory comments, has to be lowercase for serverside reasons 🌮